src/store/modules/ModalsModule.ts

```diff
@@ -38,6 +55,21 @@ export default {
             state.isDownloadModModalOpen = true;
         },
 
+        openErrorModal: function(
```

Comment:
I feel like this does more than just opens the modal so the name could be a bit misleading, would it make sense to just call this something like `handleError`?